workspaces/liferay-partner-workspace/client-extensions/liferay-partner-site-initializer/site-initializer/object-definitions/lead.json

```diff
@@ -0,0 +1,30 @@
+{
+	"accountEntryRestricted": true,
+	"accountEntryRestrictedObjectFieldName": "accountExternalReferenceCode",
+	"externalReferenceCode": "LeadObject",
```

Comment:
I am wondering if we should append the object that are using the proxy with "Proxy" So it would be LeadProxy. Similar pattern to how we did LeadSF

Reply:
Makes sense.

Reply:
Done!